coldfront/core/portal/templates/portal/authorized_home.html

```diff
@@ -115,115 +115,151 @@ <h3>My BRC Cluster Projects &raquo;</h3>
       <div class="row">
         <div class="col-lg-6 mt-2">
           <h4>Savio</h4>
-          <hr>
           {% if savio_projects %}
-            <ul class="list-group">
-              {% for project in project_list %}
-                {% if project.name in savio_projects %}
-                  <li class="list-group-item">
-                    <a href="{% url 'project-detail' project.pk %}">
-                      <i class="fa fa-folder fa-lg" aria-hidden="true"></i>
-                      {{ project.name }}
-                    </a>
-                    {% if project.needs_review %}
-                      <a href="{% url 'project-review' project.pk %}">
-                        <span class="badge badge-warning">Needs Review</span>
-                      </a>
-                    {% endif %}
+              <table class="table table-sm">
+                  <thead>
+                    <tr>
+                      <th scope="col" class="text-nowrap">Project Name</th>
+                      <th scope="col" class="text-nowrap">Cluster Access Status</th>
```

Comment:
I believe we had wanted to re-use the wording "Project Access on Cluster" from the Project Detail view to be explicit.